bin/hdfcoinc/pycbc_fit_sngls_binned

```diff
@@ -35,14 +35,17 @@ stat_dict = {
     "new_snr"       : events.newsnr,
     "effective_snr" : events.effsnr,
     "snr"           : lambda snr, rchisq : snr,
-    "snronchi"      : lambda snr, rchisq : snr / (rchisq ** 0.5)
+    "snronchi"      : lambda snr, rchisq : snr / (rchisq ** 0.5),
+    "blue_snr"      : events.bluesnr,
```

Comment:
i dont think we want to have to answer to 50 people as to why 'blue snr' makes sense. prefer to start with something rather more wonky like 'newsnr_sgveto' ?